src/app/Fake.BuildServer.TeamCity/TeamCity.fs

```diff
@@ -226,11 +232,11 @@ module TeamCity =
 
     /// Sends a warning message.
     let internal warning message =
-        TeamCityWriter.sendToTeamCity "##teamcity[message text='%s' status='WARNING']" message
+        TeamCityWriter.sendToTeamCity "##teamcity[message text='%s' status='WARNING']" (TeamCityWriter.encapsulateSpecialChars message)
```

Comment:
I think escaping is done by sendToTeamCity already (arguably a bit unexpected....)